dev/crypto.md

```diff
@@ -80,22 +82,41 @@ below specifies each prefix (in quotation marks):
        - "aS": A _Settlement_.
 
 
-## Hash Function
+## Hash Functions
 
+### SHA512-256
 Algorand uses the [SHA-512/256 algorithm][sha] as its primary
 cryptographic hash function.
 
 Algorand uses this hash function to (1) commit to data for signing and
 for the Byzantine Fault Tolerance protocol, and (2) rerandomize its
 random seed.
 
+### SUBSET-SUM
+Algorand uses [SUBSET-SUM algorithm][sumhash] which is a quantum-resilient hash function.
+This function is used by the [Merkle Keystore](merklekeystore) to commit on
+ephemeral public keys. It is also used to create Merkle trees for the StateProofs. 
 
 ## Digital Signature
 
+### ED25519
+
 Algorand uses the [ed25519][ed25519] digital signature scheme to sign
 data.
 
 
+### FALCON
+
+Algorand uses a deterministic version of [falcon scheme][falcon]. Falcon is quantum resilient and a SNARK friendly digital signature scheme used to sign in StateProofs. 
+
+The library defines the following sizes:
```

Comment:
Can you please provide the other parameters:
modulus q, standard deviation sigma, ...

as well as an estimate of the resulting security (128 bits of security?)

If parameters are unusual, can you please provide a link to a security analysis?

Reply:
Can you provide details about encoding of public key, private key and signature? (big/little endian and so on)?

Reply:
Same as above. There will be a link to the full detailed spec created by Algorand's crypo research team.